app/utils/helpers.ts

```diff
@@ -46,3 +47,15 @@ export function findProjectsBySlug(projects: Project[], slugs: string[]) {
     })
     .filter((p): p is Project => !!p);
 }
+
+/**
+ * Take a contract address and return the corresponing token abbreviation
+ * @param address Contract address of the token
+ * @param tokens List of tokens in the app
+ * @returns {string}
+ */
+export const toTokenAbbreviation = (address: string, tokens: Token[]) => {
+  const tokenMatch = tokens.find((t) => t.contractAddress === address);
+  invariant(tokenMatch, "Could not match token contract address to a token");
```

Comment:
lets not do an invariant and let this return undefined. No reason to crash if we can display a token symbol (like the now gone USDC)